src/util/ReactUtils.ts

```diff
@@ -260,6 +262,9 @@ const SVG_TAGS: string[] = [
 
 const isSvgElement = (child: any) => child && child.type && _.isString(child.type) && SVG_TAGS.indexOf(child.type) >= 0;
 
+export const isDotProps = (dot: LineDot | AreaDot): dot is DotProps =>
+  dot && typeof dot === 'object' && 'cx' in dot && 'cy' in dot && 'r' in dot;
```

Comment:
Nit: It might be nice going forward to start smaller files with smaller scope to move away from generic collect all files. 

This function for example could live in a separate file. It seems unrelated to all other functions. 

But that really is just personal preference.

Reply:
Sure - to me at the time it seemed meta enough that 'ReactUtil' fit, but in reality its more of a TypeScript util or even a specific Dot util